client/src/meta/env.meta.js

```diff
@@ -1,6 +1,8 @@
 const meta = require('./meta-util')
 
-const staticUrl = process.env.STATIC_URL || 'localhost:8080'
+const staticUrl = process.env.STATIC_URL || 'http://localhost:8080'
 
-module.exports = meta.json({ staticUrl })
+const apiUrl = process.env.API_URL || 'http://localhost:8080/api'
```

Comment:
TODO later to https

Reply:
Localhost will always be `http://` https can be added through `STATIC_URL` environment variable